app/assets/javascripts/models/beta/pastIteration.js

```diff
@@ -0,0 +1,25 @@
+import httpService from '../../services/httpService';
+import changeCase from 'change-object-case';
+import PropTypes from 'prop-types';
+import { deserialize } from './story';
+
+export const getStories = async (projectId, startDate, endDate) => {
+    const params = serialize({startDate, endDate});
+    const { data } = await httpService
+        .get(`/project_boards/${projectId}/iterations`, { params });
+    return data.stories.map(story => deserialize(story.story));
+};
+
+export const pastIterationPropTypesShape = PropTypes.shape({
```

Comment:
I could not find where this is being used. :thinking:

Reply:
Good catch! It's supposed to be used for the sprint component's `propTypes` but i forgot about it